ServiceProviderBot/Bot/Prompts/LessThanOrEqualPromptValidator.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Bot.Builder.Dialogs;
+using System.Threading.Tasks;
+
+namespace ServiceProviderBot.Bot.Prompts
+{
+    public static class LessThanOrEqualPromptValidator
```

Comment:
Used to validate openings are less than total, rather than having to manually catch that. Now the prompt will automatically retry if this validator fails